torch/overrides.py

```diff
@@ -42,7 +42,7 @@
 ]
 
 @functools.lru_cache(None)
-def get_ignored_functions() -> Set[Callable]:
+def get_ignored_functions() -> Set[Any]:
```

Comment:
```suggestion
def get_ignored_functions() -> Set[Callable]:
```

Reply:
Not every member in the list returned by `get_ignored_functions` is a Callable. For instance, `torch.cpp` is a module.

Reply:
Can you try removing that and seeing if the tests still pass?

Reply:
You mean `mypy` or the python/C++ tests?

Reply:
Modules shouldn't be returned here, because they don't need to be ignored in the first place.

Reply:
Okay! Let me try

Reply:
I'd remove anything that's not callable here and post a comment with the test failures. Then add those failures to the ignores.

Reply:
Formally, this should be a `Set[Callable]`. If it isn't there's a bug _somewhere_, either an actual bug, or one in type inference.

Reply:
The list above contains some properties (i.e. `torch.has_cuda`) and those are typed as just the return type and not as a callable.

Reply:
What about changing those to `property.__get__`?

Reply:
Ok, I'll resolve this conversation. It seems that `Callable` is working just fine with latest changes. :D